docs/src/faq.md

```diff
@@ -0,0 +1,32 @@
+# FAQ
+
+
+## Breaking Changes
```

Comment:
It would be cool to not write the changelog here but rather "why do i have error x and how to i fix it".

So one thing could be "my invocation of buffrs login is broken" and the answer would be that we renamed `--url` to `--registry`

Reply:
Very good!